src/templates/next/components/internal/Footer/Footer.tsx

```diff
@@ -96,7 +96,7 @@ const SocialMediaSection = ({
 }: Pick<FooterProps, "socialMediaLinks">) => {
   return (
     <div className="flex flex-col gap-5">
-      <h3 className={Heading[5]}>Reach us</h3>
+      <h3 className={Heading['subhead-1']}>Reach us</h3>
```

Comment:
:( not a fan of this. would rather create a new obj called `Subheading`, then reference as `Subheading[1]` But since @dcshzj is implementing typography soon, not a prior to fix in this particular pr

Reply:
okie! didn't touch this